src/main/kotlin/prison/model.kt

```diff
@@ -9,15 +9,10 @@ fun prisonModel(model: Model) {
   model.setImpliedRelationshipsStrategy(
       CreateImpliedRelationshipsUnlessAnyRelationshipExistsStrategy())
 
-  val nomis = NOMIS(model)
```

Comment:
What do you think about initialising all model objects here instead of `workspace.kt`? I'd be confused if I'd see Nomis on the model without seeing it at the same level as others.

Having it in the same file would make it more prominent that there's a transition in progress.

Reply:
Hi @sldblog I'd rather refactor these models out of here in a subsequent PR. Having the model -> relationship -> view steps in `workspace.kt` makes it more cohesive, and allows us to iterate over the classes rather than defining them in multiple places. 

I don't mind doing the legwork to refactor the other classes out after this is merged. We could also:
- Add a comment into this file to say not to use it, or
- Move the model definitions up into `workspace.kt` and remove this file. It's only a few lines, so should be quite obvious there.

I've added a comment to this file to make it clearer and hopefully reduce confusion.

Reply:
👍 Sounds good. Hopefully, we can remove the comments soon (I don't like comments :neckbeard:)